const x = 42;

try {
  console.log(nkglhjkb);
} catch (err) {
  console.error(err);
} finally {
  console.log('Hello');
}
